tests/continuous/.gitignore

```diff
@@ -0,0 +1 @@
+*.DS_Store
```

Comment:
I think you only need the one `.gitignore` in the parent directory and that this can be removed